lib/ical/time.js

```diff
@@ -165,25 +165,45 @@ class Time {
     }
 
     let zone;
+    let zoneId;
 
     if (aValue[19] && aValue[19] === 'Z') {
-      zone = 'Z';
+      zone = Timezone.utcTimezone;
     } else if (prop) {
-      zone = prop.getParameter('tzid');
+      zoneId = prop.getParameter('tzid');
+
+      if (prop.parent) {
+        if (prop.parent.name === 'standard' || prop.parent.name === 'daylight') {
+          // Per RFC 5545 3.8.2.4 and 3.8.2.2, start/end date-times within
+          // these components MUST be specified in local time.
+          zone = Timezone.floating;
```

Comment:
Where is `Timezone.floating` defined?

Reply:
This may be an error on my part; it should probably be `Timezone.localTimezone`.